eventkit_cloud/api/views.py

```diff
@@ -1132,6 +1135,17 @@ def partial_update(self, request, id=None, *args, **kwargs):
         # administrator of the current group or there is an attempt to end up with no administrators
 
         if not self.useradmin(group,request):
+            user = User.objects.all().filter(username=request.user.username)[0]
```

Comment:
You don't need .all() here. 
https://docs.djangoproject.com/en/2.0/topics/db/queries/#retrieving-specific-objects-with-filters